nimdoc/testproject/expected/subdir/subdir_b/utils.html

```diff
@@ -16,7 +16,7 @@
 <link href='https://fonts.googleapis.com/css?family=Source+Code+Pro:400,500,600' rel='stylesheet' type='text/css'/>
 
 <!-- CSS -->
-<title>Module utils</title>
+<title>utils — Nim v0.19.1</title>
```

Comment:
Gah, so we need to touch this test then for every release? That sucks.

Reply:
Yeah, that's how I did it (manually changed it).

I can remove it so we have only `<title>utils</title>`, like it was in the first commit.

Reply:
So ... yeah, no version then or add some wildcard support for the nim doc tester.

Reply:
Ok, the new commit has no version, and uses "Imports" as before (making it a very small PR :)). Merge?